src/main/java/net/onelitefeather/antiredstoneclockremastered/service/RedstoneClockService.java

```diff
@@ -169,7 +173,7 @@ private void sendNotification(final Player player, final Location location) {
     private void breakBlock(@NotNull Location location) {
         Block block = location.getBlock();
         if (this.dropItems) {
-            var drops = block.getDrops();
+            var drops = block.getDrops(SILK_TOUCH_PICKAXE);
```

Comment:
Can you double-check if the durability gets reduced ?

Reply:
<img width="569" alt="image" src="https://github.com/user-attachments/assets/57c74fe6-1aa0-4f0c-ba6e-77bb41c9f772">
The method only checks what items **would** drop if the block were to be destroyed with a tool 👍🏻

Reply:
But using a tool consume durability from the tool

Reply:
I've put it to the test and called the method 10,000 times, it still gave the same result.

`@Override
    public void onEnable() {
       ItemStack pickaxe = new ItemStack(Material.DIAMOND_PICKAXE);
       pickaxe.addEnchantment(Enchantment.SILK_TOUCH, 1);
       Location location = new Location(Bukkit.getWorlds().getFirst(), 0, 0, 0);
       Block block = location.getBlock();
       Collection<ItemStack> itemDrops = block.getDrops(pickaxe);
       int counter = 0;

       for (int i = 0; i < 10000; i++) {
           var newItemDrops = block.getDrops(pickaxe);
           if (!newItemDrops.equals(itemDrops)) {
               throw new RuntimeException("ERROR: Result of getDrops changed");
           }
           itemDrops = newItemDrops;
           counter++;
       }
       getLogger().info("Ran tool test " + counter + " times without error");
    }`

Reply:
Looks good then to me. An diamond pickaxe have just 1561 durability